erroranalysis/requirements-dev.txt

```diff
@@ -1,4 +1,4 @@
-pytest==7.0.1
+pytest<=7.1.2
```

Comment:
Can we strong pin this to 7.1.2? <= can pull an older version.

Reply:
I can't pin to 7.1.2 directly because older versions of python are not supported.  7.1.2 only supports python >= 3.7:
https://pypi.org/project/pytest/7.1.2/
![image](https://user-images.githubusercontent.com/24683184/168160237-c0cfcd40-6ba7-4cc2-9f37-41892eda3f62.png)

Reply:
we are still testing python 3.6, perhaps we need to remove that at some point soon